package.json

```diff
@@ -27,6 +27,9 @@
     "test": "cross-env NODE_ENV=test npm run test:build && jasmine-browser-runner runSpecs --config=test/support/jasmine-browser.json --browser=headlessChrome"
   },
   "main": "./dist/main.js",
+  "files": [
+    "dist/"
```

Comment:
Files included in package by default are `package.json`, `README.md`, and `LICENSE`.
https://docs.npmjs.com/cli/v7/commands/npm-publish

Should we include demo/bin here?